gridstatus/tests/test_miso.py

```diff
@@ -108,6 +108,7 @@ def test_lmp_date_range(self, market):
         Markets.REAL_TIME_HOURLY_PRELIM,
     )
     def test_get_lmp_historical(self, market):
+        # Prelim data only goes back 4 days
```

Comment:
These tests [did fail](https://github.com/kmax12/gridstatus/actions/runs/9784853404/job/27016653393) when the error was introduced but since it constantly fails now, I didn't wait for it to finish before merging. This specific test passes now although there are still others that do not in `test_miso`.